openforcefield/topology/topology.py

```diff
@@ -861,7 +861,7 @@ def __init__(self, other=None):
         elif isinstance(other, FrozenMolecule):
             self.from_molecules([other])
         elif isinstance(other, OrderedDict):
-            self.initialize_from_dict(other)
+            self._initialize_from_dict(other)
```

Comment:
It appears that neither of these functions actually exist. Wonder how these got here in the first place. Fascinating!

It's definitely better to refer to these as private methods if they're yet-to-be-implemented.